src/overlay.js

```diff
@@ -346,29 +386,80 @@
         },
 
         /**
-         * Changes the location and placement of the overlay.
+         * Changes the overlay settings.
```

Comment:
Looks like this function now supports a lot more options than are documented... should the docs be updated?